package/CHANGELOG

```diff
@@ -13,10 +13,10 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
---/--/-- tyler.je.reddy, kain88-de, richardjgowers, manuel.nuno.melo, 
+--/--/15 tyler.je.reddy, kain88-de, richardjgowers, manuel.nuno.melo,
```

Comment:
We should be able to get 0.13.0 out, there's quite a lot of good stuff in there.